solara/components/togglebuttons.py

```diff
@@ -86,26 +149,72 @@ def Page():
     reactive_value = solara.use_reactive(value, on_value)  # type: ignore
     children = [solara.Button(label=str(value)) for value in values] + children
     values = values + [_get_button_value(button) for button in children]  # type: ignore
-    index, set_index = solara.use_state_or_update(values.index(reactive_value.value) if reactive_value.value is not None else 0, key="index")
+    # When mandatory = True, index should not be None, but we are letting the front-end take care of setting index to 0 because of a bug
+    # (see https://github.com/widgetti/solara/issues/282)
+    # TODO: set index to 0 on python side (after #282 is resolved)
+    index, set_index = solara.use_state_or_update(values.index(reactive_value.value) if reactive_value.value is not None else None, key="index")
+    # When mandatory = True, index should not be None, but we are letting the front-end take care of setting index to 0 because of a bug
+    # (see https://github.com/widgetti/solara/issues/282)
+    # TODO: set index to 0 on python side (after #282 is resolved)
+    index, set_index = solara.use_state_or_update(values.index(reactive_value.value) if reactive_value.value is not None else None, key="index")
 
     def on_index(index):
         set_index(index)
-        value = values[index]
-        reactive_value.set(value)
+        if mandatory:
+            value = values[index]
+            reactive_value.set(value)
+        else:
+            if index is not None:
+                value = values[index]
+                reactive_value.set(value)
+            else:
+                reactive_value.set(None)
 
     return cast(
         reacton.core.ValueElement[v.BtnToggle, T],
-        rv.BtnToggle(children=children, multiple=False, mandatory=True, v_model=index, on_v_model=on_index, dense=dense, class_=class_, style_=style_flat),
+        rv.BtnToggle(children=children, multiple=False, mandatory=mandatory, v_model=index, on_v_model=on_index, dense=dense, class_=class_, style_=style_flat),
     )
 
 
+@overload
+@overload
 @solara.value_component(None)
 def ToggleButtonsMultiple(
     value: Union[List[T], solara.Reactive[List[T]]],
+    values: List[T] = ...,
+    children: List[reacton.core.Element] = ...,
+    on_value: Union[Callable[[Optional[List[T]]], None], None] = ...,
```

Comment:
```suggestion
    on_value: Union[Callable[[List[T]], None], None] = ...,
```
I think for mandatory is True, it should be a list of T